tests/src/main/java/org/hibernate/beanvalidation/tck/util/ArtifactDumper.java

```diff
@@ -117,8 +117,7 @@ private static void addClassesForPackage(String packageName, ArrayList<Class<?>>
 			if ( file.getAbsolutePath().endsWith( ".class" ) ) {
 				// removes the .class extension
 				String className = file.getPath().substring( 0, file.getPath().length() - 6 );
-				className = className.replace( "/", "." );
```

Comment:
Good catch. But I think we should still replace the `File.separator` with a `.`.

Reply:
Yeah, I'm not sure it's safe enough. Considering the fully qualified class name seems safer to me. I believe that's what the current code is doing.